talisker/logs.py

```diff
@@ -466,6 +466,22 @@ def logfmt_value(self, v):
 
         return v
 
+    def string_needs_quoting(self, v):
+        all_numeric = True
+        decimal_count = 0
+        for c in v:
+            if c in ' ="\\':
+                return True
+            if c not in '012345679.':
```

Comment:
err, what happened to 8?

You might want to consider `if not (c.isdigit() or c == '.'):`

What about other number like values e.g. '1E07', or 'Inf', 'NaN' etc.?